media/js/privacy/privacy-day.js

```diff
@@ -5,10 +5,106 @@
 $(function() {
     'use strict';
 
-    // Hide the download button from Firefox users
-    if (isFirefox()) {
-        $('#download').hide();
-    } else {
-        $('#principles').hide();
+    var pager = Mozilla.Pager.pagers[0];
+
+    // scroll to top of pager when switching tabs so all
+    // tab content is visible after switching tabs
+    pager.$container.on('changePage', function() {
+
+        // Get the current document scroll position
+        var scrollPos = $(document).scrollTop();
+
+        // Get the offset of the top of the pager section
+        var pagerTopPos = $('.pager').first().offset().top;
+
+        // If we're scrolled passed the header, jump back up
+        // to the tabs when the tab is switched
+        if (scrollPos > pagerTopPos) {
+            $('html, body').scrollTop($('.pager').first().offset().top);
+        }
+    });
+
+    // enable sticky tab nagivation
+    $('#button-nav-wrapper').waypoint('sticky');
+
+    // Get the text of an element excluding any children elements
+    jQuery.fn.justtext = function() {
+        return $(this) .clone()
+                .children()
+                .remove()
+                .end()
+                .text()
+                .trim();
+    };
+
+    var trackClick = function (gaArgs, href, event) {
+        if (event.metaKey || event.ctrlKey) {
+            // Open link in new tab
+            gaTrack(gaArgs);
+        } else {
+            event.preventDefault();
+            gaTrack(gaArgs, function() { window.location = href; });
+        }
+    };
+
+    var getCurrentTab = function () {
+        var currentTab = 'None';
+        var el = $('#tips-nav-direct a.selected');
+        if ($(el).length) {
+            return el.justtext();
```

Comment:
Is it expected that this will return the translated text, depending on the locale of the page? Would GA events be better captured using an english value, i.e. based on the href?